scripting/include/retakes.inc

```diff
@@ -47,9 +47,10 @@ forward void Retakes_OnPreRoundEnqueue(Handle rankingQueue, Handle waitingQueue)
  * (rather than using the Retakes_SetRoundPoints native)
  *
  * @param rankingQueue a priority queue (see include/priorityqueue.inc)
+ * @param waitingQueue a priority queue (see include/priorityqueue.inc)
  * @noreturn
  */
-forward void Retakes_OnPostRoundEnqueue(Handle rankingQueue);
+forward void Retakes_OnPostRoundEnqueue(Handle rankingQueue, Handle waitingQueue);
```

Comment:
Based on https://github.com/splewis/csgo-retakes/blob/master/scripting/retakes.sp#L222
the comment for waitingQueue should reference include/queue.inc